doc/dev/GETTING_STARTED.md

```diff
@@ -86,9 +86,14 @@ mkdir build
 
  * Open git bash
 
+Check the VCPKG baseline in `vcpkg.json` at the root of f3d source directory.  
+It should be `c9140a3b500812ad3206317885860d9553b93f13` (please let us know if it's not)
```

Comment:
What does it mean to "check the VCPKG baseline" ?

Keep in mind this guide is for complete beginners. If VCPKG doesnt work out of the box, we may consider modifying the guide to build VTK manually.

Reply:
> modifying the guide to build VTK manually

Definitely not. I don't want to write a guide for every single dependency.

> Check the VCPKG baseline in `vcpkg.json` at the root of f3d source directory.

I'll be more clear, but that's something a beginner can check easily.

Reply:
Let me know if it's better now.

Reply:
On windows, VTK is the only needed dependency afaik

Reply:
And plenty of other optional dependencies. I still think VCPKG is way more beginner-friendly.

Reply:
I agree